tests/data/jsons/menu-name.json

```diff
@@ -13,6 +13,7 @@
         "echo",
         "A"
       ],
+      "activate": false,
```

Comment:
Why do we need these test changes?

Reply:
`activate` only works when the prefix is a `conda` environment, which is not the case for these shortcuts. This causes the activator command to fail. For `.bat` files, that command fails when the shortcut is executed, which we don't do for the tests.

For `.env` files, however, the activator is called at install time, which causes the installation of the shortcut to fail. I actually like that behavior more, even though it is currently limited to Windows.

Reply:
Hm, I see. I wonder if we check for `conda` in the target prefix before we write the shortcut logic, but that will require API changes so `conda` can pass that boolean, because I'm not sure `menuinst` should be the one checking. Thoughts?